app/models/post.rb

```diff
@@ -13,7 +13,7 @@ def post_counter
     user.increment!(:posts_count)
   end
 
-  def five_recent_comments(user_id, posts_id)
-    Comment.where(user_id: user_id, posts_id: posts_id).order(created_at: :desc).limit(5).all
+  def five_recent_comments(user_id, post_id)
+    Comment.where(user_id: user_id, post_id: post_id).order(created_at: :desc).limit(5).all
   end
```

Comment:
- [ ] OPTIONAL: This was part of the previous milestone, I noticed that you are expecting a user id and post id in this function, this is JS way of doing things, in rails you don't have to do that, the below code will work fine:

 ```
 def five_recent_comments
    comments.order(created_at: :desc).limit(5)
  end
```